### ISSUE TITLE
Feature Request: Allow multiple root folders to be added to the sidebar

### ISSUE THREAD

**Issue by sahil48** [Zettlr/Zettlr#3]
For example, Dropbox, OneDrive, and Documents Folder can all be available in the Sidebar.

**Comment by nathanlesage** [comment]
Good idea, I had planned this as well. But first I have to fix several issues concerning the watching of the files (there's still a mess)
